authz/sdk_end2end_test.go

```diff
@@ -418,23 +417,34 @@ func (s) TestSDKFileWatcher_ValidPolicyRefresh(t *testing.T) {
 
 	// Rewrite the file with a different valid authorization policy.
 	valid2 := sdkTests["AllowsRpcEmptyDenyMatchInAllow"]
-	if err := ioutil.WriteFile(file, []byte(valid2.authzPolicy), os.ModePerm); err != nil {
-		t.Fatalf("ioutil.WriteFile(%q) failed: %v", file, err)
+	if err := os.WriteFile(file, []byte(valid2.authzPolicy), os.ModePerm); err != nil {
+		t.Fatalf("os.WriteFile(%q) failed: %v", file, err)
 	}
-	// Wait 2 seconds for background go routine to read the updated files.
-	time.Sleep(2 * time.Second)
+	// Wait 30 ms for background go routine to read the updated files.
+	time.Sleep(30 * time.Millisecond)
 
 	// Verifying authorization decision.
 	_, err = client.UnaryCall(ctx, &pb.SimpleRequest{})
```

Comment:
This needs to be done in the loop or else you aren't actually retrying.

Reply:
Oh right! Updated code.